website/content/docs/k8s/deployment-configurations/vault/data-integration/connect-ca.mdx

```diff
@@ -13,6 +13,8 @@ This topic describes how to configure the Consul Helm chart to use TLS certifica
 Consul allows using Kubernetes auth methods to configure Connect CA.
 This allows for automatic token rotation once the renewal is no longer possible.
 
+~> **Note:** Do not use Vault v1.11.0+ as Consul's Connect CA provider — the intermediate CA will become unable to issue the leaf nodes required by service mesh and all actions (if using auto-encrypt or auto-config, and using mTLS for server-to-server communication). If you are already using Vault 1.11+ as a Connect CA, refer to the [Knowledge Base article]() for more information about the underlying cause and recommended workaround.
```

Comment:
I had been wondering about whether we needed anything about the Consul k8s integration with Vault as a general secrets backend, but it looks like you've already covered that here. I initially thought it might need to go on the overview page too (https://developer.hashicorp.com/consul/docs/k8s/deployment-configurations/vault), but it looks like you have to opt into the different kinds of secrets, and this page covers opting into using Vault as the CA... so I think that's all that we need?